pkg/lib/cockpit-po-plugin.js

```diff
@@ -100,12 +100,19 @@ module.exports = class {
             const parsed = gettext_parser.po.parse(fs.readFileSync(po_file), 'utf8');
             delete parsed.translations[""][""]; // second header copy
 
+            const manifest_file = path.resolve(srcdir, "pkg/shell/manifest.json");
+            const manifest = JSON.parse(fs.readFileSync(manifest_file, 'utf8'));
+            // po files use short version of locales - instead of `he-il` it uses `he`
+            const rtl_langs = manifest.rtl_locales.map(lang => lang.split("-")[0]);
+            const dir = rtl_langs.indexOf(parsed.headers.language) > -1 ? "rtl" : "ltr";
```

Comment:
style nitpick: `list.contains(item)` is easier to read, and harder to get wrong, it does not need that `> -1` thing